File: Selenium/Activities/Activity_16.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;

public class Activity_16 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
        driver.get("https://v1.training-support.net/selenium/dynamic-attributes");
        driver.manage().window().maximize();
        System.out.println("The pagetitle is: " + driver.getTitle());
        driver.findElement(By.cssSelector("input[class$='username']")).sendKeys("saurav");
        driver.findElement(By.cssSelector("input[class$='password']")).sendKeys("Saurav");
        driver.findElement(By.xpath("//label[text()='Confirm Password']/following::input[1]")).sendKeys("Saurav");
        driver.findElement(By.xpath("//input[@type='email']")).sendKeys("saurav@ok.com");
        driver.findElement(By.xpath("//button[@onclick='signUp()']")).click();
        wait.until(ExpectedConditions.textToBePresentInElementLocated(By.xpath("//div[@id='action-confirmation']"), "Thank you for registering, saurav"));
        System.out.println(driver.findElement(By.xpath("//div[@id='action-confirmation']")).getText());
        driver.quit();
    }
}

File: Selenium/Activities/Activity_16.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/dynamic-attributes")
    print("The page title is: ", driver.title)
    driver.maximize_window()
    mywait = WebDriverWait(driver, 10)
    driver.find_element(By.CSS_SELECTOR, "input[class$='username']").send_keys("nomad")
    driver.find_element(By.CSS_SELECTOR, "input[class$='password']").send_keys("None")
    driver.find_element(By.XPATH, "//label[text()='Confirm Password']/following::input[1]").send_keys("None")
    driver.find_element(By.XPATH, "//input[@type='email']").send_keys("hariom@ok.com")
    driver.find_element(By.XPATH, "//button[@onclick='signUp()']").click()
    mywait.until(EC.text_to_be_present_in_element((By.XPATH, "//div[@id='action-confirmation']"), "Thank you for registering, nomad"))
    print(driver.find_element(By.XPATH, "//div[@id='action-confirmation']").text)
    driver.quit()